openedx_caliper_tracking/processor.py

```diff
@@ -110,9 +114,13 @@ def __call__(self, event):
             if getattr(settings, 'CALIPER_DELIVERY_ENDPOINT') and getattr(settings, 'CALIPER_DELIVERY_AUTH_TOKEN'):
                 deliver_caliper_event(transformed_event, event.get('event_type'))
 
+            if settings.FEATURES.get('ENALBE_KAFKA_FOR_CALIPER') and hasattr(settings, 'CALIPER_KAFKA_SETTINGS'):
```

Comment:
One more thing, please document this flag in our readme.

Reply:
Once the documentation of ucsd/edx-platform documentation is updated, it would be done in that step.